lib/util.go

```diff
@@ -16,7 +16,7 @@ import (
 
 func isArchiveFile(filePath string) bool {
 	fileExtension := filepath.Ext(filePath)
-	if fileExtension == ".br" || fileExtension == ".bz2" || fileExtension == ".zip" || fileExtension == ".gz" || fileExtension == ".lz4" || fileExtension == ".sz" || fileExtension == ".xz" || fileExtension == ".zst" || fileExtension == ".tar" || fileExtension == ".rar" {
+	if fileExtension == ".br" || fileExtension == ".bz2" || fileExtension == ".zip" || fileExtension == ".gz" || fileExtension == ".lz4" || fileExtension == ".sz" || fileExtension == ".xz" || fileExtension == ".zst" || fileExtension == ".tar" || fileExtension == ".rar" || fileExtension == ".tgz" {
 		return true
 	}
 	return false
```

Comment:
Love the thought that this should be comprehensive! I actually looked again into the archiver (v3) library and found [this method](https://github.com/mholt/archiver/blob/v3.5.1/archiver.go#L441) `ByExtension` that actually does what we are looking for! Might I suggest the following:

```diff
func isArchiveFile(filePath string) bool {
-	fileExtension := filepath.Ext(filePath)
-	if fileExtension == ".br" || fileExtension == ".bz2" || fileExtension == ".zip" || fileExtension == ".gz" || fileExtension == ".lz4" || fileExtension == ".sz" || fileExtension == ".xz" || fileExtension == ".zst" || fileExtension == ".tar" || fileExtension == ".rar" || fileExtension == ".tgz" {
-		return true
-	}
-	return false
}

func isArchiveFile(filePath string) bool {
+	_, err := archiver.ByExtension(filePath)
+	return err == nil
}